python/ray/train/backend.py

```diff
@@ -21,6 +21,8 @@
 
 T = TypeVar("T")
 
+C = TypeVar("C")
```

Comment:
Maybe we can rename this into something more description, e.g:
```suggestion
EncodedCheckpointData = TypeVar("EncodedCheckpointData")
```

or

```suggestion
EncodedData = TypeVar("EncodedData")
```

Reply:
Done!